ios/Runner/AppAuthHandler.swift

```diff
@@ -177,9 +210,16 @@ class AppAuthHandler {
         OIDAuthorizationService.perform(registrationRequest) { (response, error) in
             
             if let error = error {
+                
+                if self.isTimeoutError(error) {
+                    os_log("The connection timed out. ")
+                    self.invokeErrorHandler(code: self.RUNTIME_ERROR, errorMessage: "Timeout to register service")
+                    return
+                }
+                
                 os_log("Failed registration client %{error}@", log: .default, type: .error, error.localizedDescription)
                 self.setAuthState(nil) // Clear local store - if already stored
-                self.invokeErrorHandler(errorMessage: "Failed to register service: \(error.localizedDescription)")
+                self.invokeErrorHandler(code: self.FAILED_REGISTER, errorMessage: "Failed to register service: \(error.localizedDescription)")
```

Comment:
Is there a not "localized" description? Given we don't translate these (yet), mixing languages seems weird. Ditto all the others.

Reply:
The 'localized' is not really localized, but user readable. We could make it accessible in a user facing message? To keep it in logs is more important. 
It could be hard to manual translate every kind of error - Will this help a user?

Reply:
It might help us to help a user.

Reply:
Then the deeph should be fixed: On network level (After validating the username) there are three network calls: 
- Service discovery
- Register Client 
- Request Auth
- Get token

- Token Refresh

This should be discussed.

Reply:
Sorry, I cannot follow your comment.